### DISCUSSION TITLE
Trigger Mac Expose (mission control)

### DISCUSSION THREAD

**Discussion by NorfeldtKnowit** [rxhanson/RectanglePro-Community#494] [Ideas]
Been setting BetterTouch (BT) to trigger the mac expose with one of the additional mouse buttons I got.

![mac expose](https://upload.wikimedia.org/wikipedia/en/6/68/MacOS_Mission_Control_screenshot.png)

I'm just wondering if Rectangle Pro could have this feature as well?

(Another thing BT allows me to do is setup a gesture so I can close a window with a four finger touch gesture)

**Comment by NorfeldtKnowit** [comment]
I found another solution with https://github.com/noah-nuebling/mac-mouse-fix - just keeping the idea open, then you can decide to close it.

**Comment by rxhanson** [comment]
Thanks for the feedback. This is a little outside of what I would want to add into Rectangle Pro, since it's primarily about window management. This is possible in one of my other apps, [Multitouch](https://multitouch.app), but Multitouch doesn't have everything that's in Rectangle Pro. I don't mind leaving this open, in case something changes around this idea.

**Comment by NorfeldtKnowit** [comment]
thank you @rxhanson for making me aware about multitouch - think your reasoning for splitting the features across different apps and not make a swish knife app that does too much makes good sense. 

I'm closing the discussion since I found what I'm looking for. Thanks
